tests/Tokenizer/Analyzer/CommentsAnalyzerTest.php

```diff
@@ -465,4 +465,67 @@ enum E {}
             ',
         ];
     }
+
+    /**
+     * @dataProvider provideReturnStatementCases
+     */
+    public function testReturnStatement(string $code, bool $allowReturn, bool $expected): void
+    {
+        $tokens = Tokens::fromCode($code);
+        $index = $tokens->getNextTokenOfKind(0, [[T_COMMENT], [T_DOC_COMMENT]]);
+        $analyzer = new CommentsAnalyzer();
+
+        self::assertSame($expected, $analyzer->isBeforeStructuralElement($tokens, $index, $allowReturn));
+    }
+
+    public static function provideReturnStatementCases(): iterable
```

Comment:
please declar phpdoc for return iterable.
`@return iterable<string, array{string, bool, bool}>`

Reply:
Done